source/github-helpers/index.test.ts

```diff
@@ -91,7 +91,6 @@ test('isUsernameAlreadyFullName', () => {
 	assert.isTrue(isUsernameAlreadyFullName('john-wdoe', 'John W. Doe'));
 	assert.isTrue(isUsernameAlreadyFullName('john-doe-jr', 'John Doe Jr.'));
 	assert.isTrue(isUsernameAlreadyFullName('nicolo', 'Nicolò'));
-	assert.isTrue(isUsernameAlreadyFullName('johnwashere', 'John'));
```

Comment:
Oh I see what my intention was: don't show a name that doesn't add anything. However this function didn't do what I expected.